zipline/testing/core.py

```diff
@@ -761,7 +763,8 @@ class tmp_assets_db(object):
     """
     _default_equities = sentinel('_default_equities')
 
-    def __init__(self, equities=_default_equities, **frames):
+    def __init__(self, url='sqlite://', equities=_default_equities, **frames):
```

Comment:
I think you can equivalently have this be `'sqlite:///:memory:'`, which might be clearer for the reader.